### ISSUE TITLE
Image ids are incorrect in API output

### ISSUE THREAD

**Issue by chrismytton** [mysociety/popit#470]
e.g. on http://congresistas-gt.popit.mysociety.org/api/v0.1/persons/5367ae9d7ee559063fbc6b6e the object in the `images` array has the following id property:

```
"_id": {
  "id": "Sr.~ö5?Bed\u0007ú",
  "_bsontype": "ObjectID"
}
```

Where it should actually just be a string:

```
"_id": "53722e7ef6353f42656407fa"
```

Additionally it would be useful if each object in the images array had a `url` property pointing to the image.

**Comment by chrismytton** [comment]
Still seeing this issue when uploading a new image to a newly created document.

**Comment by chrismytton** [comment]
The issue here is that the image is added to the `images` array by the UI. In the UI the `Person` schema has an embedded `Image` schema, so when an image is added to the `images` array it gets an `_id` property which is a mongo ObjectId. However, when the API then gets the requested person out of the database it *doesn't* have an embedded image schema, it just sees `images` as an array of plain object, in this case it seems that mongoose doesn't do the conversion to a string representation of an ObjectId, which is why we we see the strange `_bsontype` property.

So tl;dr the root of this issue is the fact that we have two sets of schemas accessing the same database table (one in the UI, on in the API) but representing them in different ways. The fix therefore is to consolidate the schemas so there is One True Source of data modelling.
